core/drivers/crypto/caam/utils/utils_mem.c

```diff
@@ -203,13 +77,13 @@ static void *mem_alloc(size_t size, uint8_t type)
 
 	if (type & MEM_TYPE_ALIGN) {
 		cacheline_size = read_cacheline_size();
-		if (ADD_OVERFLOW(alloc_size,
-				 ROUNDUP(alloc_size, cacheline_size),
-				 &alloc_size))
+		if (ROUNDUP_OVERFLOW(alloc_size, cacheline_size, &alloc_size))
```

Comment:
Is this `ROUNDUP_OVERFLOW()` needed?

Reply:
`memalign()` should be OK with any size.

Reply:
I will discard it